src/wwwroot/template-editor/adapterService.js

```diff
@@ -27,6 +28,18 @@
       overrideUploadFiles: overrideUploadFiles
     };
 
+    var langKey = null;
+    init();
+    
+    function init() {
```

Comment:
Could you declare these functions after _returning service instance_ in the same way that we are doing with the other functions?